packages/cra-template/scripts/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -xe
```

Comment:
Unfortunately we can't use shell scripts as part of `test` scripts since our tests also have to run on Windows (and we can't assume WSL or git bash). I was thinking something more along the lines of the temp dir creation in `packages/fluentui/projects-test/src/createReactApp.ts`.

Reply:
Ah gotcha. Thanks for the heads up! Lemme redo this then!

Reply:
Done! Have a look and lemme know what you think!